test/unit/api/compress/compress_wide.cpp

```diff
@@ -1,9 +1,12 @@
 #include "unit/api/compress/compress_test.hpp"
 
-TTS_CASE_WITH( "Check compress behavior"
-        , eve::test::simd::all_types
-        , tts::generate(tts::ramp(1))
-        )
+#ifdef SPY_SIMD_IS_RISCV_FIXED_RVV
+using types = tts::filter<rvv_compress_filter, eve::test::simd::all_types>::type;
+#else
+using types = eve::test::simd::all_types;
+#endif
```

Comment:
I'd suggest: 

put this in comoress tests

call it simd_types_for_compress

Reply:
Done